src/tree.ts

```diff
@@ -210,7 +210,7 @@ export class DirectoryNode<Node extends BaseNode> extends BaseNode {
 }
 
 export class SourceFileNode extends BaseNode {
-  constructor(readonly prefix: string, readonly sourcePath: string, readonly filePath: string) {
+  constructor(readonly prefix: string, readonly sourcePath: string, readonly filePath: string, private _language?: string) {
```

Comment:
This will sound nitpicky, and I apologize, but the member variables in these classes don't start with a leading underscore.  Can you remove it?

Reply:
i don't mind changing it, but shouldn't it have the underscore?

> Prefix private members/methods with an underscore and write them _withCamelCase ([coding style](https://vector-of-bool.github.io/docs/vscode-cmake-tools/development.html))

the private members of the other classes in that file start with the underscore.
 e.g. `private _subdirs = new Map<string, DirectoryNode<Node>>();`

Reply:
Ah.  The other classes that declared member variables in the constructor signature didn't use underscores, and that's what threw me off.

It's fine though.  We will be changing the style guide when we migrate the docs off of the old website (which we don't have access to update).  We'll need to do a wholesale update when we do this, and we can update this at that time.